XenAdmin/Controls/NetworkingTab/NetworkList.cs

```diff
@@ -946,6 +948,9 @@ public void UpdateDetails()
                     Pif == null ? Messages.NONE : Pif.LinkStatusString();
                 MacCell.Value = Pif != null && Pif.IsPhysical() ? Pif.MAC : Messages.SPACED_HYPHEN;
                 MtuCell.Value = Network.CanUseJumboFrames() ? Network.MTU.ToString() : Messages.SPACED_HYPHEN;
+                SriovCell.Value = Pif == null || Pif.sriov_logical_PIF_of == null || Pif.sriov_logical_PIF_of.Count == 0

```

Comment:
As I had also said on PR  #1954, please create a PIF.IsSrIovLogicalPIF() function ans use it here as well.